### PR TITLE
fixed for v45

### PR DESCRIPTION
**Pull request by Commander-Cat101**

### CODE REVIEW

File: UI.cs [hunk 1]
```diff
@@ -50,6 +50,7 @@ public static bool ValidTiers(IReadOnlyCollection<int> tiers) =>
     
     public static void SetUpLevelInput()
     {
+        var slider = levelselect.AddSlider(new Info("Slider"), 0, 0, 5, 0.1f, new Vector2(100, 100));
```
Comment (GrahamKracker) [inline UI.cs:53]: what is this for

File: Merging/Algorithm.cs [hunk 2]
```diff
@@ -115,6 +116,10 @@ public static Il2CppReferenceArray<Model> MergeBehaviors(Il2CppReferenceArray<Mo
 
     private static bool IsType<T>(this Type typ)
     {
+        if (typeof(T).IsEnum)
```
Comment (GrahamKracker) [inline Merging/Algorithm.cs:119]: why was this added exactly, in theory the IsAssignableFrom should have been correct?

**Comment by GrahamKracker** [comment]
bump version

**Review by GrahamKracker — CHANGES_REQUESTED** [review]

File: Merging/Algorithm.cs (hunk 2 above)
Comment (Commander-Cat101) [reply Merging/Algorithm.cs:119]: it threw AccessViolationException so I updated it to use UltimateCrosspathing fixed method

File: UI.cs (hunk 1 above)
Comment (Commander-Cat101) [reply UI.cs:53]: no clue tbh, it was already there, idk why it shows as new

File: Merging/Algorithm.cs (hunk 2 above)
Comment (GrahamKracker) [reply Merging/Algorithm.cs:119]: ok i guess its a funny il2cpp enum bug yippee

**Review by GrahamKracker — APPROVED** [review]
